packages/vite/src/plugins/plugin.ts

```diff
@@ -235,6 +240,8 @@ function getOutputs(
     'dist'
   );
 
+  const isBuildable = !!build || existsSync(join(projectRoot, 'index.html'));
```

Comment:
@jaysoo we still want that, right? Or don't we, due to what @Coly010 had said "some vite plugins like qwikVite will alter the entry points and therefore there won’t be an index.html present"?

Reply:
`build` is coming from `viteConfig` but, `viteConfig` is still using `loadConfigFromFile` and i don't see any mappings to check for `rollup.build`? is there a different approach now?

Reply:
It looks like @ndcunningham has mentioned that `build` should only be truthy if the project is buildable, which vite works out from `loadConfigFromFile`. 

So I think this might be ok? 

But I'm not sure the fallback is needed. Does `vite` not already create `build` if `index.html` is present and it assumes it as an entry point by default?

Reply:
Vite will try to find the entry point from `index.html` by default unless you or a plugin provides it. I'll double-check this logic, but from what I recall it is fine.